setup.py

```diff
@@ -53,8 +53,6 @@ def desc():
         "Flask-Babel>=1, <3",
         "Flask-Login>=0.3, <0.5",
         "Flask-OpenID>=1.2.5, <2",
-        # SQLAlchemy 1.4.0 breaks flask-sqlalchemy and sqlalchemy-utils
-        "SQLAlchemy<1.4.0",
```

Comment:
nit: While I assume 1.4 will be followed by 2.0, maybe we should keep the restriction but move it one minor up, like
```
 "SQLAlchemy<1.5",
```

Reply:
I understand your concern, mixed thoughts here. The cap responsibility should come from flask-sqlalchemy. But probably makes sense given FAB huge "dependency" on SQLAlchemy